vuepress/docs/next/docs/consume/operator-intro.md

```diff
@@ -50,17 +50,17 @@ determined.
 expected to contain two files: tls.key and tls.crt.
   - If a key pair is found in the folder specified, it will revert to the keypair found.
   - If a key pair is not found the Entando Operator will evaluate the value of the `ENTANDO_USE_AUTO_CERT_GENERATION`.
-    If that property is set to `true` the Entando Operator will assume that the cluster has been configured with a valid CA and leave it to
+    If that property is set to `true`, the Entando Operator will assume that the cluster has been configured with a valid CA and leave it to
     the Ingress controller to generate its own certificates.
 
-[Click here for tutorials on creating secrets and setting up TLS in your Entando Apps.](../../tutorials/getting-started/openshift-install-by-operator-hub.md)
+[Click here for tutorials setting up TLS in your Entando Apps.](../../tutorials/getting-started/openshift-install-by-operator.md)
 
 ## Database Deployment
 
 Some Entando components include the ability to select a database management system (DBMS) when deploying the component.
 
 - The DBMS field in Entando Custom Resources can be one of mysql, oracle, postgresql or embedded.
-  - **IMPORTANT!** -- If embedded is selected for a component only 1 replica for the component can be created.
+  - **IMPORTANT!** -- If embedded is selected for a component, only one replica for the component can be created.
   - **IMPORTANT!** -- Oracle instances are not supported for automatic deployment in a container. You must create an Oracle instance or use an existing instance and configure it as an [external database](../../tutorials/devops/external-db.md) for your Entando app
```

Comment:
or use an existing instance and configure it as an [external database](../../tutorials/devops/external-db.md) for your Entando app -> configure an existing instance as an [external database](../../tutorials/devops/external-db.md) for your Entando app. (shorten + period)